Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.7
+
+RUN apt-get update
+RUN apt-get install --yes --no-install-recommends wget git maven less
```

Comment:
Nit: his should be on the same line as the apt-get update.
https://docs.docker.com/v17.09/engine/userguide/eng-image/dockerfile_best-practices/#apt-get

Also the packages should be listed one by line in alphabetical order

Reply:
Good points. https://github.com/openzipkin-contrib/apache-release-verification/pull/7/commits/5106cdfa62e8ccbe71b088b7500a5576115f22e6 addresses this, please resolve if happy :)

Reply:
> please resolve if happy

I don't have a resolve button, probably because I'm not a member. But yeah looks good to me